### PR TITLE
Added Empty Views

### PR DESCRIPTION
**Pull request by opticod**
Added #340 . Here are some empty views. I have also added empty views for tracks, speakers, sessions and  microlocations.
screenshot_20160411-025208
screenshot_20160411-025233
screenshot_20160411-025243

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java [hunk 1]
```diff
@@ -47,12 +48,15 @@
 
     private SearchView searchView;
 
+    private TextView noMicrolocationsView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java]: Codacy Issue found: [Perhaps 'noMicrolocationsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630555/issues/source?bid=3233850&fileBranchId=3233897#l51)

File: app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 2]
```diff
@@ -46,6 +47,8 @@
 
     private SearchView searchView;
 
+    private TextView noSessionsView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: Codacy Issue found: [Perhaps 'noSessionsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630496/issues/source?bid=3233850&fileBranchId=3233897#l50)

File: app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java [hunk 3]
```diff
@@ -45,6 +46,8 @@
 
     private SearchView searchView;
 
+    private TextView noSessionsView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java]: Codacy Issue found: [Perhaps 'noSessionsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630502/issues/source?bid=3233850&fileBranchId=3233897#l49)

File: app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java [hunk 4]
```diff
@@ -71,13 +72,16 @@
 
     private int sortType;
 
+    private TextView noSpeakersView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java]: Codacy Issue found: [Perhaps 'noSpeakersView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630556/issues/source?bid=3233850&fileBranchId=3233897#l75)

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java [hunk 5]
```diff
@@ -55,13 +55,16 @@
 
     private boolean flagSocial = false;
 
+    private TextView noSessionsView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Codacy Issue found: [Perhaps 'noSessionsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630500/issues/source?bid=3233850&fileBranchId=3233897#l58)

File: app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java [hunk 6]
```diff
@@ -42,6 +43,8 @@
 
     private SwipeRefreshLayout swipeRefreshLayout;
 
+    private TextView noSponsorsView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: Codacy Issue found: [Perhaps 'noSponsorsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630557/issues/source?bid=3233850&fileBranchId=3233897#l46)

File: app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java [hunk 7]
```diff
@@ -55,12 +56,15 @@
 
     private SearchView searchView;
 
+    private TextView noTracksView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java]: Codacy Issue found: [Perhaps 'noTracksView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630558/issues/source?bid=3233850&fileBranchId=3233897#l59)

**Comment by opticod** [comment]
@juslee made the changes :smile:

**Comment by opticod** [comment]
issue is still open? any hints for closing? @the-dagger

**Comment by the-dagger** [comment]
Hey, we'll be redesigning these parts of the app so no reason of merging this. 
Cheers

**Comment by shivenmian** [comment]
rekt

**Pull request closed without merging**